.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+## Filing a bug report
+
+We love to receive bug reports (we're always trying to make ShellJS more
+stable). If you've found a bug, please follow these steps:
+
+ - Search for any issues that may have been created already. We often receive
+   duplicates, and cutting down on this is helpful. If someone else has already
+   reported it, please ping that issue thread.
+ - Let us know your version of NodeJS (`node -v`), your version of ShellJS (from
+   `package.json`), your OS, and any other useful information.
+ - Give an example ShellJS command to reproduce the error.
```

Comment:
I said "Example ShellJS command to reproduce the error:" Pretty sure that'll do the trick. Let me know if you think otherwise.